### DISCUSSION TITLE
override the "__" laravel translate method

### DISCUSSION THREAD

**Discussion by niketnarola** [NIT-Administrative-Systems/dynamic-forms#460] [Q&A] [answered]
Before installation of the "dynamic-forms" packages it's working well. and while I've installed the "dynamic-forms" package and it generates the error.

![image](https://github.com/NIT-Administrative-Systems/dynamic-forms/assets/69069187/0fa4546e-4271-4a44-b729-5d807c050555)

It's override the laravel "__(Dashboard)" language translate method with your lodash.php file function. 

![image](https://github.com/NIT-Administrative-Systems/dynamic-forms/assets/69069187/fb2e18ff-32c2-4af6-8dee-24570cab7ba0)

Please check and share your suggestion for it.

**Comment by tuursw** [comment]
same problem was also mentioned in the issues section
https://github.com/NIT-Administrative-Systems/dynamic-forms/issues/452

**Reply by niketnarola** [reply]
Do you have a solution for the same?

**Comment by dxnter** [comment, marked as answer]
This issue has been resolved in [`v3.0.0`](https://nit-administrative-systems.github.io/dynamic-forms/setup/upgrading/#v300)
